### DISCUSSION TITLE
A change detection, zone.js, zoneless, local change detection, and signals story 📚 - justangular.com

### DISCUSSION THREAD

**Discussion by giscus** [eneajaho/just-angular#21] [General]
# A change detection, zone.js, zoneless, local change detection, and signals story 📚 - justangular.com

How change detection will be replaced by signals in Angular and why it doesn't make sense to talk call it change detection anymore.

https://justangular.com/blog/a-change-detection-zone-js-zoneless-local-change-detection-and-signals-story/

**Comment by baldmaan** [comment]
great article, thanks!

**Comment by trminhphu79** [comment]
Many thanks for your post.
I just have 1 question.
Does the term "refresh a view" mean re-render only the part of the template whose data binding changed? or the whole component?

**Comment by quocle79** [comment]
This is a great article that helps explains how Angular detect changes and which component/view to render. The accompanied playground is excellent also. Thanks.
